core-build/gulp-core-build-sass/src/SassTask.ts

```diff
@@ -205,77 +196,38 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
           result.styles.toString()
         ];
         if (result.sourceMap && !this.buildConfig.production) {
-          const encodedSourceMap: string = Buffer.from(result.sourceMap.toString()).toString('base64');
-          generatedFileLines.push(...[
+          const encodedSourceMap: string = Buffer.from(result.sourceMap.toString())
+                                                 .toString('base64');
```

Comment:
Even though I wrote a bunch of it a few years ago, I'm kinda not a fan of this floating syntax anymore. Is it too long as a single line?

Reply:
Not necessary. Reverted back.